custom/icds_reports/utils/data_pull_ap_rj.sql

```diff
@@ -196,13 +196,22 @@ COPY(
     SUM(CASE WHEN month='2019-11-01' THEN num_awcs_conducted_vhnd ELSE 0 END) as conducted_vhnd_nov,
```

Comment:
@mkangia / @calellowitz  I added the thr_eligible along with thr 21 days. the percentage can be done via excel or via python.
Also, this thr data is not present for month April, may, June in this table. But I tried pulling from other tables and those were just very expensive. I think the best way would be to add these month data first to this table and then fetch it. This also might not be the best idea because we are filling the data just for a data pull. But I can't think any other way.